_build/sharpmake/src/generate_settings.cs

```diff
@@ -167,7 +167,7 @@ public override string AsString(string key)
 
         foreach (string val in project_vals.Value)
         {
-          sb.AppendLine($"    {ElementType}{{ {val}, \"{project_vals.Key}\" }},");
+          sb.AppendLine($"    {ElementType}{{ {val}, L\"{project_vals.Key}\" }},");
```

Comment:
Do we need this in the project generation files?
There is going to be a lot of overhead to keep track of where we need to introduce UNICODE and where we don't need to introduce it. 
I would try and limit it to one place ( maybe it can get converted to a UNICODE structure somewhere? )

Reply:
https://github.com/New-Creta/Rex/issues/281